config/rabe-fs-backup.timer

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Run RaBe Backup Service timer daily
+
+[Timer]
+# Run once per day
```

Comment:
To help users understand what the [daily](https://www.freedesktop.org/software/systemd/man/systemd.time.html#Calendar%20Events) expression means, supplement the comment by adding _run once per day **at midnight**_